src/stories/Library/Forms/checkbox/checkbox.scss

```diff
@@ -54,7 +54,13 @@
   }
 
   &--hide-visually {
```

Comment:
"@extend .hide-visually !optional;" doesnt seem to work, perhaps because of the order the scss are imported?

Reply:
We should have a mentally note on that. Could be a code smell.